IntelliTect.Multitool/Security/ClaimsPrincipalExtensions.cs

```diff
@@ -16,7 +16,7 @@ public static class ClaimsPrincipalExtensions
         /// <param name="principal">The <see cref="ClaimsPrincipal"/> to find a user ID for.</param>
         /// <returns>A <see cref="string"/>, or null if the <see cref="ClaimsPrincipal"/> doesn't contain a <see cref="ClaimTypes.NameIdentifier"/>.</returns>
         /// <exception cref="ArgumentNullException">principal is null.</exception>
-        public static string GetUserId(this ClaimsPrincipal principal)
+        public static string? GetUserId(this ClaimsPrincipal principal)
```

Comment:
There are a few more null checks that should be done with NRT enabled here.

For example:
```c#
Claim claim = principal.FindFirst(ClaimTypes.NameIdentifier);
```
`claim` should either be nullable or the null check removed (I suspect the former).
Similar comments on the other null checks in this file

Reply:
Hopefully I improved these!